### ISSUE TITLE
Review and improve mobile layout of course pages

### ISSUE THREAD

**Issue by ragesoss** [WikiEducationFoundation/WikiEduDashboard#1283]
Make sure all the key data on a course page is accessible via mobile, such as number of edits for each user.

**Comment by Vinit-Dantkale** [comment]
Hey @ragesoss , its old issue Can I work on it? if it is not resolved.

**Comment by ragesoss** [comment]
Yes, please do! There's still quite a few things to improve about the mobile layout.

**Comment by Manasa2850** [comment]
@ragesoss can I work on this? I have solved the layout issues of the timeline page here https://github.com/WikiEducationFoundation/WikiEduDashboard/pull/3649
I would like to work on the other pages too.

**Comment by shipcy** [comment]
Can I work on this?

**Comment by Manasa2850** [comment]
@ragesoss can I create separate issues for each bug and work on it?

**Comment by ragesoss** [comment]
@Manasa2850 yes, that would be great. @shipcy if you also create a separate issue for a specific aspect of the mobile layout that you're working on, then you can work on it as well.
